### ISSUE TITLE
Formulaic assemblicator multiblock

### ISSUE THREAD

**Issue by KatherineLee318** [mekanism/Mekanism-Feature-Requests#476]
### Describe the feature you'd like

just a more advanced formulaic assemblicator that combines the qio importer, exporter, and redstone adapter into one machine.

it should also use a formula card that contains the recipe and how many need to be made

### Additional context

its mostly to help cut down on any lag that would be caused by having tons of the formulaic assemblers running with all the importers, exporters, and redstone adapters

**Comment by pupnewfster** [comment]
No idea if/when this will actually happen, but I definitely do like the idea of some more compact/automatic way to handle the stock keeping styled auto crafting. Given while I don't think on demand crafting really fits mek, making it easier to setup a crafter that keeps x amount of things without having to do any redstone would be a pretty nice QoL upgrade for the formulaic assemblicator

**Comment by KatherineLee318** [comment]
yeah, it's kind of a pain having to go in and set it up for even just a few items that are constantly needed not to mention you can't really put them right next to one another due to three sides being completely used by power and the import/exporter with maybe the Redstone adapter.



though i do think it should require a good bit of power to run
